api/docs/code_reviews.dox

```diff
@@ -79,7 +79,29 @@ can select a particular reviewer if desired.
 
 \section sec_commit_messages Commit Messages
 
-Commit messages should include an initial title line separated from the body of the message by a blank line.  The title line should be a concise summary and should start with the issue number followed by a colon which is followed by the description of the issue fix.  If the issue is fixed by more than one commit, the title should instead start with the issue number followed by a space and a short issue description, then a colon, and then a description of this particular fix, with a reference to the issue at the bottom for automatic linking in Github.  In either case, the body of the message should elaborate on the contents of the commit using complete sentences.
+A pull request's final commit message body upon merging comes from the pull request's
+description, which is the first comment box on the pull request's Conversation tab.
```

Comment:
s/comment box/conversation comment/?

"comment box" may be confused to mean the box where one types.

Reply:
It is such a box: it says "[derekbruening](https://github.com/derekbruening)
commented
[9 hours ago](https://github.com/DynamoRIO/dynamorio/pull/7715#issue-3634555496)" on existing ones, and new ones say "Add a comment".  Do you mean "comment box" makes you think only new ones and not existing ones?